generator/main.ts

```diff
@@ -56,9 +64,16 @@ async function main([out_dir, ...idl_files]) {
     });
   });
 
+  let interface_names = new Array<String>();
   for (let idl_file of idl_files) {
-    await generateInterface(env, idl_file, out_dir);
+    const parsedData = webidl.parse(await file.read(path.resolve(idl_file)));
+    const idl_fragments: idls.Fragments = new idls.Fragments(parsedData);
+    await generateInterface(env, idl_file, out_dir, idl_fragments);
+    if (idl_fragments.definitions[0] instanceof idls.InterfaceImpl)
```

Comment:
could you add "for" keyword to iterate definitions.
(refer to https://github.com/lunchclass/bacardi/pull/89/files#diff-94ecd868e07f21781a35a525c94f7de9R38)

Reply:
thanks for your review :) as your recommend, I added code to generate multi interfaces with iterating definitions.
actually, I should change generateInterface also, but it will be fixed with your pull request(#89). so I didn't modify it.